external/Makefile.tools

```diff
@@ -58,15 +58,15 @@ axe:
 	cp $(AXE_DIR)/axe $(BIN_DIR)
 
 gnu:
-	rm -rf $(GNU_DIR)/riscv-newlib
 	@cd $(TOP); git submodule update --init --recursive $(GNU_DIR)
 	cd $(GNU_DIR); \
-		rm -rf riscv-newlib; \
-		git clone -b bsg_manycore_newlib https://github.com/bespoke-silicon-group/bsg_newlib_dramfs.git riscv-newlib; \
```

Comment:
Briefly, can you explain how our Linux gnu toolchain gets built and used?

Reply:
We clone the toolchain, apply the `newlib.patch` which changes the url and branch of `riscv-newlib` and applies a small Makefile change in the toolchain. Then we sync the submodule and build.

Reply:
Yeah, that’s the newlib chain. I’m asking about the Linux chain, that’s separate, no?

Reply:
I don't think we build that chain here.